src/components/menubar/component.tsx

```
@@ -43,20 +42,24 @@ export function Menubar({ isOpen, onClose, children }: ModalProps) {
     };
 
     const handleResetSettings = () => {
-        browser.runtime
+        chrome.runtime
             .sendMessage({
                 type: "RESET_SETTINGS",
             })
             .then((response) => {
                 if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)